lib/public/views/QcFlags/details/forDataPass/QcFlagDetailsForDataPassPage.js

```diff
@@ -12,36 +12,83 @@
  * or submit itself to any jurisdiction.
  */
 
+import { detailsList } from '../../../../components/Detail/detailsList.js';
 import errorAlert from '../../../../components/common/errorAlert.js';
 import { deleteButton } from '../../../../components/common/form/deleteButton.js';
+import { markdownDisplay } from '../../../../components/common/markdown/markdown.js';
+import { frontLink } from '../../../../components/common/navigation/frontLink.js';
+import { LabelPanelHeaderComponent } from '../../../../components/common/panel/LabelPanelHeaderComponent.js';
+import { PanelComponent } from '../../../../components/common/panel/PanelComponent.js';
+import { tooltip } from '../../../../components/common/popover/tooltip.js';
 import spinner from '../../../../components/common/spinner.js';
-import { qcFlagDetailsComponent } from '../qcFlagDeailsComponent.js';
-import { h } from '/js/src/index.js';
+import { qcFlagDetailsConfiguration } from '../detailsConfiguration.js';
+import { h, iconWarning } from '/js/src/index.js';
 
 /**
  * Render QC flag details for data pass page
  *
  * @param {Model} model The overall model object.
  * @returns {Component} The details page
  */
-export const QcFlagDetailsForDataPassPage = ({ qcFlags: { detailsForDataPassModel, forDataPassOverviewModel } }) => h('', {
-    onremove: () => {
-        forDataPassOverviewModel.reset();
-    },
-}, [
-    h('.flex-row.justify-between.items-center', [
-        h('h2', 'QC Flag Details'),
-        h('.pv3', detailsForDataPassModel.deleteResult.match({
-            Loading: () => deleteButton(null, 'Processing...'),
-            Success: () => deleteButton(null, 'Processed!'),
-            Failure: () => deleteButton(() => detailsForDataPassModel.delete()),
-            NotAsked: () => deleteButton(() => detailsForDataPassModel.delete()),
-        })),
-    ]),
-    detailsForDataPassModel.qcFlag.match({
-        Success: () => h('.flex-row.g3.flex-grow', [qcFlagDetailsComponent(detailsForDataPassModel)]),
-        Loading: () => spinner(),
-        NotAsked: () => errorAlert([{ title: 'No data', detail: 'No QC flag data was asked for' }]),
-        Failure: (errors) => errorAlert(errors),
-    }),
-]);
+export const QcFlagDetailsForDataPassPage = ({ qcFlags: { detailsForDataPassModel, forDataPassOverviewModel } }) => {
+    const { qcFlag: remoteQcFlag, deleteResult, dataPass: remoteDataPass } = detailsForDataPassModel;
+    const qcFlagDetailsDispayConfigration = qcFlagDetailsConfiguration(detailsForDataPassModel);
```

Comment:
You don't need to pass the model anymore right?
Also: the file name `detailsConfiguration` is wrong

Reply:
right, fixed, name changed